src/toast/todmap/pointing.py

```diff
@@ -281,3 +285,315 @@ def exec(self, data):
         data[npix_name] = 12 * self._nside ** 2
 
         return
+
+
+class OpMuellerPointingHpix(Operator):
+    """
+    Operator which generates I/Q/U healpix pointing weights.
+
+    Given the individual detector pointing, this computes the pointing weights
+    assuming that the detector is a linear polarizer followed by a total
+    power measurement.  An optional dictionary of calibration factors may
+    be specified.  Additional options include specifying a constant cross-polar
+    response (eps) and a rotating, perfect half-wave plate.  The timestream
```

Comment:
We should update the docstring for this operator since it is no longer a perfect half-wave plate.